src/content/en/updates/2018/11/webcomponents-v0-deprecations.md

```diff
@@ -0,0 +1,102 @@
+we're deprecating Shadow DOM v0, Custom Element v0 and HTML Imports in Chrome 73.
+
+[Intent to Deprecate](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/h-JwMiPUnuU/sl79aLoLBQAJ)
+
+
+## Motivation
+
+Shadow DOM v0, Custom Elements v0, and HTML Imports were launched in 2014, but
+they did not get adopted by other browser engines. Instead, Shadow DOM v1,
+Custom Elements v1, and ES modules are widely adopted by various browser engines
+today. Chrome has shipped Shadow DOM V1 / Custom Elements V1 in 2016 and ES
+modules in 2017.
+
+- Shadow DOM v0: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960) &#124;
+ [Chromium Bug](https://crbug.com/671907)
+- Custom Elements v0: [Chromestatus Tracker](https://www.chromestatus.com/feature/4642138092470272)  &#124;
+ [Chromium Bug](https://crbug.com/660759)
+- HTML Imports: [Chromestatus Tracker](https://www.chromestatus.com/feature/4507242028072960)  &#124;
+[Chromium Bug](https://crbug.com/766694)
+
+### Why at once?
+
+Since then, we also found that in many real-world use cases, Shadow DOM V0,
+Custom Elements V0, HTML Imports were used together, and instead of getting each
+ of them migrated respectively, getting all of them migrated to today’s standards
+ at once, or at least use the polyfill would make it less troublesome, easier to
+ migrate, rather than fighting with every edge case of combination of polyfill
+  and native implementation.
+
+## Migration support
+If you are using this features, please consider migrating to the new API or
+upgrading your Polymer library.
+Use `--disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports` for
+ testing if your site works without those APIs.
+
+## Remove Shadow DOM v0
+`Element.createShadowRoot()` will be removed. Instead we have `Element.attachShadow()`
+to create Shadow Root.
+If your existing components don't use distribution, replacing with
+`Element.attachShadow({mode:"open"})` might work.
+Otherwise, Shadow DOM v1 is not backward compatible to v0.
+See a [document](https://hayato.io/2016/shadowdomv1/) for further comparing and
+our [document](/web/fundamentals/web-components/shadowdom) describes v1 functions and usage.
+
+
+## Remove Custom Elements v0
+`document.registerElement()` will be replaced by  `window.customElements.define()`.
+If you are calling `document.registerElement('my-button', MyButton)` with class name rather than prototype,
+`window.customElements.define('my-button', MyButton)` should work.
+Our [document](/web/fundamentals/web-components/customelements) describes v1 functions and usage.
+
+
+## Remove HTML Imports
+
+HTML Imports will be removed. Similar functionarity will be served as HTML Modules.
+Though HTML Module is still under [spec discussion](https://github.com/w3c/webcomponents/issues/645)
+and we don't have any direct replacement for HTML Imports yet, we're deprecating this for healthy web:
```

Comment:
we're -> Chrome is throughout. Although we (Chrome team) is writing this article, we still like to come across like we're discussing what the "browser vendors" are doing.